web/src/routes/Dashboard/Dashboard.js

```diff
@@ -444,7 +447,10 @@ function mapDispatchToProps(dispatch) {
       return deactivateApp(appId, cellId, dispatch)
     },
     fetchEntryPoints: (cellIdString) => {
-      return dispatch(fetchEntryPoints.create({ cellIdString, payload: null }))
+      return dispatch(fetchEntryPoints.create({ cellIdString, payload: { All: null } }))
+    },
+    fetchGoals: (cellIdString, addresses) => {
+      return dispatch(fetchGoals.create({ cellIdString, payload: { Specific: addresses } }))
```

Comment:
I wonder if you think if it may have made more sense to keep the fetch{entry_type}s the same, and then only add the FetchOptions to inner_fetch{entry_type}s? It seems like in most cases we are fetching all of that type, and that fetching a specific list is more of an edge case (at least in the case of acorn).